app/models/ability.rb

```diff
@@ -84,11 +84,11 @@ def permissions_for_finance_admin_user
     can :charge_adjust, :all
     can :write_off_large, WasteCarriersEngine::FinanceDetails
     can :view_certificate, WasteCarriersEngine::Registration
-    can :record_worldpay_missed_payment, :all
+    can :record_online_missed_payment, :all
```

Comment:
Previously `online_payment` has been directly substituted for `worldpay`. I agree that `record_online_payment_missed_payment` might be a bit clumsy - but is it clearer?

Reply:
Fair point, for consistency this should be aligned with the use of `online_payment` elsewhere. Although `payment` is already part of the name. Perhaps `record_online_payment_missed` would be both more consistent and less clumsy?